cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/dataset/input/BasicPartitionedFileSetInputContext.java

```diff
@@ -80,7 +127,7 @@ private PartitionKey getPartitionKey(URI inputPathURI) {
     }
     throw new IllegalArgumentException(
       String.format("Failed to derive PartitionKey from input path '%s' and path to key mapping '%s'.",
-                    inputPath, pathToPartitionMapping));
+                    inputPathURI, pathToPartitionMapping));
```

Comment:
this will definitely get huge if the mapping contains 30k partitions. Better not log the whole map.